app/components/primer/alpha/select_panel.pcss

```diff
@@ -0,0 +1,3 @@
+.SelectPanel-loadingPanel {
```

Comment:
```suggestion
.SelectPanel-loadingSpinner {
```

Reply:
I think this will still look odd if there aren't many results, but I guess it's still an improvement? Maybe we could add a preview to show how that would look (so setting the size to `medium_portrait` for example, and then only return one or two results

Reply:
@camertron - I don't know if `loadingSpinner` is totally accurate because this style also applies to the error message when the spinner is hidden. Do you still want the class name change?

Reply:
Hmm... alright let's keep `loadingPanel` then 👍